dashboard/settings.py

```diff
@@ -325,6 +325,11 @@
 # This is fixed from UDW
 UDW_FILE_ID_PREFIX = config("UDW_FILE_ID_PREFIX", default="1770000000")
 
+# Allow enabling/disabling the View options globally
+ENABLE_FILES_ACCESSED = config('ENABLE_FILES_ACCESSED', default=True, cast=bool)
```

Comment:
We're anticipating adding many more views in the future. 

I wonder if it would be better to rewrite this property as 

`VIEWS_DISABLED = config('VIEWS_DISABLED', default = '', cast= Csv())`

By default we'd have
`VIEWS_DISABLED=`

But you could have
`VIEWS_DISABLED=show_files_accessed,show_assignment_planning,show_grade_distribution`

Right? Then in the code just do an "in" check for this value to see if a specific view is disabled. And this is already the CSV excluded list. Then we don't have to add new settings for new views.

Maybe call it VIEW_EXCLUDED to be consistent with the Django admin.

Reply:
Sure

Reply:
I do prefer `VIEWS_DISABLED` over `VIEW_EXCLUDED` since it helps infer the idea that the view is fully disabled and not just globally hidden. Ex: data that is exclusively used for a disabled view should not be fetched in the cron (since it saves time and will help when certain views aren't yet supportable)